packages/langium/src/default-module.ts

```diff
@@ -69,7 +70,8 @@ export function createDefaultModule(context: DefaultModuleContext): Module<Langi
             ReferenceFinder: (services) => new DefaultReferenceFinder(services),
             GoToResolver: (services) => new DefaultGoToResolverProvider(services),
             DocumentHighlighter: (services) => new DefaultDocumentHighlighter(services),
-            RenameHandler: (services) => new DefaultRenameHandler(services)
+            RenameHandler: (services) => new DefaultRenameHandler(services),
+            CallHierarchyProvider: (services) => new DefaultCallHierarchyProvider(services),
```

Comment:
Suggestion: I don't believe the default implementation has real value for most languages, especially since it only supports one direction (incoming calls). Instead we should have an `AbstractCallHierarchyProvider` instead, which isn't injected by default.